vNext/test/Tester/Tester.csproj

```diff
@@ -57,6 +57,39 @@
       <Project>{40ee3b00-d381-485f-9c69-ff706837deed}</Project>
       <Name>OrleansTestingHost</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Orleans\Orleans.csproj">
```

Comment:
this is odd, does the Tester project already exist in vNext or why is it not showing it as an add? Not sure how come it's there, but I thought the plan was not to add it in vNext (or at least not for the core functionality tests).
Did you change you mind about putting tests that inherit from `HostedTestClusterEnsureDefaultStarted` in a separate test project that can be parallelized?